docs/reference/errors-and-warnings/NU1803.md

```diff
@@ -21,5 +21,16 @@ You may need to do one of the following:
 
 - Correct the specified url. You may have mistyped the source url. Change the url to explicitly request the `HTTPS` version of the source.
 - Work with the owner of the source and ask them to migrate to `HTTPS`.
+- If you are absolutely sure the 'HTTP' server you are connecting to is trustworthy, you may opt out this warning/error by setting `allowInsecureConnections` to "true" for this package source, in [Package source sections](..\nuget-config-file.md#package-source-sections) of NuGet.Config file. (Supported in NuGet 6.8+)
```

Comment:
```suggestion
- If you are absolutely certain the 'HTTP' server you are connecting to is trustworthy, you may opt out this warning by setting `allowInsecureConnections` to `true` for this package source, in [Package source sections](..\nuget-config-file.md#package-source-sections) of NuGet.Config file. (Supported in NuGet 6.8+)
```

We should just call it a warning for now as that's it is. 
We might even need to consider changing the code when it becomes an error?
Not sure how we want to handle it yet.

Reply:
Thanks! Changed it into an warning for now.
Not sure if we need to change the code when it becomes an error.
Checked https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings, seems errors about source starts from 13* while warnings start with 18* ? Not sure if changing the code will cause some confusion.
Perhaps @JonDouglas could help on this :)